framework/helpers/BaseArrayHelper.php

```diff
@@ -382,6 +382,24 @@ public static function map($array, $from, $to, $group = null)
     }
 
     /**
+     * Removes duplicate values from an array
+     * It's often faster than native array_unique
+     *
+     * @param string[]|int[] $array Can be only STRING or INTEGER values!
+     * @param bool $keepKeys Whether to maintain the array keys. If false, the resulting array
+     * @return array
+     */
+    public static function unique($array, $keepKeys = true)
+    {
+        if ($keepKeys === true) {
+            $result = array_flip(array_flip($array));
```

Comment:
why not put the return here already? it would save a bit of memory since you don't create a new variable.

it can also be done in line 397